src/main/ui/app/route/About.jsx

```diff
@@ -9,7 +9,7 @@ export default class About extends Component {
                     <h3>About</h3>
 
                     <p>
-                        <Link to="/indexingStatus">Indexing Status</Link>
+                        <Link className="btn btn-lg btn-primary col-md-1 col-sm-12 col-xs-12 buttonfix" to="/indexingStatus">Indexing Status</Link>
```

Comment:
We might even hide it (e.g. it's there only if you know the URL). For now leave it as is, but don't make it more obvious.